packages/superset-ui-translation/src/Translator.js

```diff
@@ -0,0 +1,29 @@
+import Jed from 'jed';
+import { sprintf } from 'sprintf-js';
+
+const DEFAULT_LANGUAGE_PACK = {
+  domain: 'superset',
+  locale_data: {
+    superset: {
+      '': {
+        domain: 'superset',
+        lang: 'en',
+        plural_forms: 'nplurals=1; plural=0',
+      },
+    },
+  },
+};
+
+export default class Translator {
+  constructor({ languagePack = DEFAULT_LANGUAGE_PACK } = {}) {
```

Comment:
does this support  `new Translator();` calls (without any object passed as config)? or maybe this would?

```javascript
{ 
  languagePack = DEFAULT_LANGUAGE_PACK, 
} = { 
  languagePack: DEFAULT_LANGUAGE_PACK,
};
```

Reply:
It works. There is also a unit test for `new Translator()`.